docs/pages/api/alert.md

```diff
@@ -13,7 +13,7 @@ filename: /packages/material-ui-lab/src/Alert/Alert.js
 ```js
 import Alert from '@material-ui/lab/Alert';
 // or
-import { Alert } from '@material-ui/lab';
+import Alert from '@material-ui/lab/Alert';
```

Comment:
@oliviertassinari I agree with the other changes but this doesn't really make sense

Reply:
^^ you are right, I did fix it on master :)